### PR TITLE
update 1.9->1.10

### PR DESCRIPTION
**Pull request by Stolidbug**
update 1.9->1.10

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,47 +18,47 @@
...
-            "Acme\\SyliusExamplePlugin\\": "src/"
+            "Acme\\SyliusExamplePlugin\\": "src/",
+            "Tests\\Acme\\SyliusExamplePlugin\\": "tests/"
         }
     },
     "autoload-dev": {
-        "classmap": ["tests/Application/Kernel.php"],
-        "psr-4": {
-            "Acme\\Tests\\SyliusExamplePlugin\\": "tests/"
-        }
```
Comment (pocky) [inline composer.json]: You need to revert this change

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -21,16 +21,12 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3]
-                symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.9.0]
+                php: [7.4]
+                symfony: [ 5.2]
```
Comment (pocky) [inline .github/workflows/build.yml]: Remove the space before "5"

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -21,16 +21,12 @@ jobs:
...
+                sylius: [~1.10.0]
```
Comment (pocky) [inline .github/workflows/build.yml]: ~1.9.0 must be present in the array

File: .github/workflows/build.yml (hunk 2 above)
Comment (Stolidbug) [reply .github/workflows/build.yml]: fait

File: .github/workflows/build.yml (hunk 3 above)
Comment (Stolidbug) [reply .github/workflows/build.yml]: fait

File: composer.json (hunk 1 above)
Comment (Stolidbug) [reply composer.json]: fait

File: src/AcmeSyliusExamplePlugin.php [hunk 4]
```diff
@@ -15,4 +15,4 @@ public function getPath(): string
     {
         return \dirname(__DIR__);
     }
-}
+}
```
Comment (pocky) [inline src/AcmeSyliusExamplePlugin.php]: Missing return line

File: src/DependencyInjection/Configuration.php [hunk 5]
```diff
@@ -9,11 +9,14 @@
...
+     */
     public function getConfigTreeBuilder(): TreeBuilder
     {
         $treeBuilder = new TreeBuilder('acme_sylius_example_plugin');
         $rootNode = $treeBuilder->getRootNode();
 
         return $treeBuilder;
     }
-}
+}
```
Comment (pocky) [inline src/DependencyInjection/Configuration.php]: Missing return line

File: tests/Application/config/bundles.php [hunk 6]
```diff
@@ -56,5 +54,5 @@
     Lexik\Bundle\JWTAuthenticationBundle\LexikJWTAuthenticationBundle::class => ['all' => true],
     Sylius\Bundle\ApiBundle\SyliusApiBundle::class => ['all' => true],
     SyliusLabs\DoctrineMigrationsExtraBundle\SyliusLabsDoctrineMigrationsExtraBundle::class => ['all' => true],
-    Symplify\ConsoleColorDiff\ConsoleColorDiffBundle::class => ['dev' => true, 'test' => true],
+
```
Comment (pocky) [inline tests/Application/config/bundles.php]: Remove this line

File: tests/Application/config/routes/sylius_admin_api.yaml [hunk 7]
```diff
@@ -1,3 +1 @@
-sylius_admin_api:
-    resource: "@SyliusAdminApiBundle/Resources/config/routing.yml"
-    prefix: /api
+
```
Comment (pocky) [inline tests/Application/config/routes/sylius_admin_api.yaml]: Remove this file

File: tests/Application/config/routes/sylius_api.yaml [hunk 8]
```diff
@@ -1,3 +1 @@
-sylius_api:
-    resource: "@SyliusApiBundle/Resources/config/routing.yml"
-    prefix: "%sylius.security.new_api_route%"
+#
```
Comment (pocky) [inline tests/Application/config/routes/sylius_api.yaml]: Remove this file

File: src/AcmeSyliusExamplePlugin.php (hunk 4 above)
Comment (Stolidbug) [reply src/AcmeSyliusExamplePlugin.php]: ok

File: src/DependencyInjection/Configuration.php (hunk 5 above)
Comment (Stolidbug) [reply src/DependencyInjection/Configuration.php]: ok

File: tests/Application/config/bundles.php (hunk 6 above)
Comment (Stolidbug) [reply tests/Application/config/bundles.php]: ok

File: tests/Application/config/routes/sylius_admin_api.yaml (hunk 7 above)
Comment (Stolidbug) [reply tests/Application/config/routes/sylius_admin_api.yaml]: ok

File: tests/Application/config/routes/sylius_api.yaml (hunk 8 above)
Comment (Stolidbug) [reply tests/Application/config/routes/sylius_api.yaml]: ok
